src/settings.js

```diff
@@ -994,6 +994,14 @@ var EXPORTED_FUNCTIONS = [];
 // [link]
 var EXPORT_ALL = false;
 
+// If true, we export the symbols that are present in JS onto the Module
+// object.
+// It only does Module['X'] = X;
+//
+// This only applies to MINIMAL_RUNTIME, where symbols aren't exported by
+// default.
```

Comment:
Should we remove this last line?  Doesn't it also apply to the normal runtime in that you can disable this?